### PR TITLE
ACA-1603: Adding verefication to ensure Load Balancer was created as …

### PR DESCRIPTION
**Pull request by Yaish25491**
…expected

### CODE REVIEW

**Comment by Yaish25491** [comment]
@nirarg @p3ck @prabinovRedhat

File: changelogs/fragments/20240710-test_azure_load_balancer_with_public_ip.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Adding testing to ensure Load Balancer was created as expected
```
Comment (nirarg) [inline changelogs/fragments/20240710-test_azure_load_balancer_with_public_ip.yml:2]:
```suggestion
  - Add more variables assertions to loadbalancer integration test to ensure it was created as expected
```

File: tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/create_and_delete_lb.yml [hunk 2]
```diff
@@ -21,12 +21,19 @@
         name: "{{ load_balancer_name }}"
         resource_group: "{{ test_resource_group }}"
       register: _loadbalancer
+     
+    - name: Creating loadbalancer_id for loadbalancer creation test
```
Comment (nirarg) [inline tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/create_and_delete_lb.yml:25]:
```suggestion
    - name: Create loadbalancer expected ID for the variable assertion
```

**Review by nirarg — APPROVED** [review]
Just small comments for wording
Other than that, looks good

File: changelogs/fragments/20240710-test_azure_load_balancer_with_public_ip.yml (hunk 1 above)
Comment (Yaish25491) [reply changelogs/fragments/20240710-test_azure_load_balancer_with_public_ip.yml]: comment on changelog updated

File: tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/create_and_delete_lb.yml (hunk 2 above)
Comment (Yaish25491) [reply tests/integration/targets/test_azure_load_balancer_with_public_ip/tasks/create_and_delete_lb.yml]: name changed

**Review by p3ck — APPROVED** [review]
LGTM

**Pull request merged**